assets/js/googlesitekit/site/index.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import { get, set } from 'googlesitekit-api';
+import {
+	FETCH_CONNECTION_INFO,
+	FETCH_RESET,
+	INITIALIZE,
+	RESET,
+	RESET_FAILURE,
+	RESET_SUCCESS,
+	RECEIVE_CONNECTION_INFO,
+	RECEIVE_CONNECTION_INFO_FAILED,
+} from './index.private';
+
+export const INITIAL_STATE = {
+	connectionInfo: null,
```

Comment:
All these references should be shortened to only `connection` / `CONNECTION` / `Connection` (without the "info" part), to be consistent with the public selector.

Reply:
Fair enough. These values should never be accessed/seen externally, but I can do that 😄

Reply:
Sure, still good to keep them consistent. :)